src/frontend/assets/favicons/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/assets/mstile-150x150.png"/>
+            <TileColor>#522785</TileColor>
```

Comment:
This value appears in 3 different files and will probably eventually get out of sync. This is bound to get out of sync. Could we generate these files and/or set the value during the build?

Reply:
- Consitant is a good idea, I guess then the repo should be updated (description here comes from Pete or ping Slack channel). Note: I don't have enough privileges to update the repo description

- Images copied from NNS-dapp. Source IC logo. Probably generated manually back then or with [realfavicongenerator](https://realfavicongenerator.net/)

- I extended the `assets.ts`. Being said, not sure I use the correct "includer" (`include_bytes`) for JSON and XML and also not sure if it supports sub-folder in the path (`""/favicons/browserconfig.xml"`). Let me know what need to be modified.

- regarding generating files or values during build? yes. Will I provide it in this PR? no. Being said, not against, feel free to automate the process.